package.json

```diff
@@ -7,8 +7,8 @@
     "clean": "rimraf lib dist",
     "build": "yarn clean && git rev-parse HEAD > git-revision.txt && yarn build:compile && yarn build:compile-browser && yarn build:types",
     "build:types": "tsc --emitDeclarationOnly",
-    "build:compile": "babel src -s -d lib --verbose --extensions \".ts,.js\"",
-    "build:compile-browser": "mkdirp dist && browserify -d browser-index.js -t [ babelify --sourceMaps=inline ] > dist/browser-matrix.js",
+    "build:compile": "babel -d lib --verbose --extensions \".ts,.js\" src",
+    "build:compile-browser": "mkdirp dist && browserify -d src/browser-index.js -p [ tsify -p ./tsconfig.json ] -t [ babelify --sourceMaps=inline ] -t uglifyify > dist/browser-matrix.js",
```

Comment:
`dist/browser-matrix.js` is minified now and we've ditched the non minified version?

Reply:
*flips table*

I hate everything about browser bundles. Will fix - we should be exporting both, I just forgot :(